src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Json.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // JSON(data:any, [format:s, [included_columns:*[Value:s]]])    
+    internal sealed class JsonFunction : BuiltinFunction
```

Comment:
I've moved this code to PFX.Json and the function isn't in the core library anymore.

Reply:
The function definition is still in the code (as it should be, like we did with ParseJSON). The implementation is the one that should be in PFx.Json.

Reply:
True. Incorrect comment on my side.
I thnk there is no action now.

Reply:
Since we removed the IsBehaviorOnly override, we need to add the check in CheckSemantics to ensure that it will only be a data flow function if binary data is not requested.